Assets/RTLTMPro/Runtime/RTLTextMeshPro.cs

```diff
@@ -103,7 +103,7 @@ public void UpdateText()
             if (originalText == null)
                 originalText = "";
 
-            if (!ForceFix)
+            if (!ForceFix && !TextUtils.IsRTLInput(originalText))
```

Comment:
We shouldn't do this

Reply:
So this is back to setting LTR/RTL based on the first letter in the string, but only if the application is set to RTL?

I'm not sure what Dani's concern is, but switching on the first character seemed to produce fewer issues that we have now. It also seems like the simplest fix for the browser address bar - which could be a URL or an Arabic search term.

If we keep ForceFix as based on app language, I think we'll have an issue where if an Arabic user enters the following:
[English word 2] [Arabic word 2] [English word 1] [Arabic word 1]
It'll appear in a non-Arabic app as:
[Arabic word 1] [English word 1] [Arabic word 2] [English word 2] 

I'm not sure what the best approach is.

Reply:
I didn't mean to add this to this PR, but that is setting it back to the way it was before.